README.md

```diff
@@ -26,20 +25,45 @@ To fix spelling or grammatical errors, make updates to either the English string
 Please **do not** submit any changes to the html files other than those within the `source` directory.
 
 ### Translating the website
-Translations help us make the software relevant for people who need it around the world. 
+
+Translations help us make the software relevant for people who need it around the world.
 
 If you would like to help us translate btcpayserver.org, please [join the team on Transifex](https://www.transifex.com/btcpayserver/btcpayserver-website/dashboard/).
 
 Translations are only handled through Transifex, please do not create a pull request for translations.
 
 For more information on contributing by translating, [check the translation guides](https://docs.btcpayserver.org/support-and-community/translate)
 
-
 ## How to build
 
-More details on this workflow and how to build the site locally to come. 
+Prerequisites:
+
+- [Node.js](https://nodejs.org/en/) >= 10
+- Create a `.env` file containing the API token for Transifex (`TRANSIFEX_TOKEN="my_api_token"`)
+
+Setup:
+
+```sh
+# Install the dependencies
+npm install
+
+# Download the translations – this is mandatory for the rest to work
+npm run init
+```
+
+Build:
+
+```sh
+# Build the site locally and start dev server
+npm start
+
+# Build the site for production
+npm run build
+npm run optimize
```

Comment:
Or just create a `build:production` task which runs both

Reply:
yes, wanted to suggest that too but saw the build already runs `build:*` so I did somehow try to describe the problem instead :D
https://github.com/btcpayserver/btcpayserver.github.io/pull/75/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R20

Reply:
I've distiguished these because they are so nicely seperable. However, you are right and I'll just add an `npm run prod` command that unifies these under the hood! :)